switch/switch_main.ml

```diff
@@ -176,9 +176,17 @@ let make_server config =
       >>= fun qs ->
       on_disk_queues := qs;
       info "Reading the redo-log from %s" redo_log_path;
-      let open Lwt_result in
-      Block.connect ("buffered:" ^ redo_log_path)
+      Block.connect ~buffered:true redo_log_path
+      >>= fun block -> begin
+        match block with
+        | `Ok block -> Lwt.return block
+        | `Error e ->
```

Comment:
> val __LOC__ : string
> __LOC__ returns the location at which this expression appears in the file currently being parsed by the compiler, with the standard error format of OCaml: "File %S, line %d, characters %d-%d".
> Since 4.02.0